docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.md

```diff
@@ -53,8 +53,7 @@ Column-level schema Expectations ensure that the individual columns within your
 
 #### Expect Column Values To Be Of Type
 
-Validates that the values within a column are of a specific data type. This is more stringent
-compared to the previous Expectation, suitable for scenarios needing strict type adherence.
+Validates that the values within a column are of a specific data type. This is useful for scenarios needing strict type adherence.
```

Comment:
**Note for reviewer** This is the first expectation on the page so it's unclear what "the previous Expectation" was meant to refer to.